main.cpp

```diff
@@ -357,25 +377,35 @@ static bool add_endpoints(Mainloop &mainloop)
 
 int main(int argc, char *argv[])
 {
+    int udp_port = 0;
     const char *uartstr = NULL;
     UartEndpoint uart{};
+    UdpEndpoint udp{};
     Mainloop mainloop{};
 
     setup_signal_handlers();
 
     log_open();
 
-    if (parse_argv(argc, argv, &uartstr) != 2)
+    if (parse_argv(argc, argv, &uartstr, &udp_port) != 2)
         goto close_log;
 
     if (mainloop.open() < 0)
         goto close_log;
 
-    if (uart.open(uartstr, opt.baudrate) < 0)
-        goto close_log;
+    if (uartstr) {
+        if (uart.open(uartstr, opt.baudrate) < 0)
+            goto close_log;
 
-    g_master = &uart;
-    mainloop.add_fd(uart.fd, &uart, EPOLLIN);
+        g_master = &uart;
+        mainloop.add_fd(uart.fd, &uart, EPOLLIN);
+    } else {
+        if (udp.open("127.0.0.1", udp_port, true) < 0)
```

Comment:
Also get the IP from parameter, this way can make compute board forward messages to host computer than host computer with mavlink-router can forward messages to another networks.